.github/workflows/publish-preview.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish package preview
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - run: corepack enable
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Build
+        run: pnpm build
+
+      - run: pnpm dlx pkg-pr-new publish --comment=off --pnpm './packages/dustkit'
```

Comment:
should we also start publishing a preview of the world package?

Reply:
yea we should. Is there a cost to using this service? or its free?

Reply:
freeee babyyyy